src/docs/Foundations/Colors/index.mdx

```diff
@@ -9,10 +9,16 @@ status:
 
 > Mozaic offers an adaptive and a scalable color system. These colors have been defined using [HSLA](#hsla) color swatches. The color palette has been divided in four categories : **Primary colors**, **Secondary colors** ,**Greys** and **Status colors**.
 
+
+<Highlight theme="warning" title="Be aware">
+
 You will discover that colors are duplicated in some cases. This may seems strange to you, but we need to think about other brands and cultures to allow them to customize the colors.
 
 For example, the success color may not be green in another country, and the primary brand color may not be usable for success with another brand.
 
+</Highlight>
+
+
 **It is why you should take a special care to use the right color variable for the right usage.**
```

Comment:
it seems strange to put this sentence alone. it's deeply linked to the content of the `be aware` section